server/mergin/sync/permissions.py

```diff
@@ -158,11 +158,25 @@ def get_user_project_role(
         return None
 
 
-def require_project(ws, project_name, permission):
-    workspace = current_app.ws_handler.get_by_name(ws)
+def is_active_workspace(workspace):
+    """
+    Checks if the given workspace is active and if the current user is an admin or anonymous.
+
+    Args:
+        workspace (Workspace): The workspace to check.
+
+    Returns:
+        bool: True if the workspace is not active and the current user is not an admin or is anonymous, False otherwise.
+    """
     if not workspace:
         abort(404)
```

Comment:
this is not bool, also we do not need such check, I assume we won't pass None in arg

Reply:
https://github.com/MerginMaps/server/pull/239/commits/b267f71681394711a0d6c30b38794efdba8a374c